CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Added support for Kafka Exporter `offset.show-all` parameter
 * Prevent removal of the `broker` process role from KRaft mixed-nodes that have assigned partition-replicas
 * Improve broker scale-down prevention to continue in reconciliation when scale-down cannot be executed
+* Support for configuring custom tiered storage plugins.
```

Comment:
I would highlight more this new feature in the CHANGELOG because it's an important one. Just as suggestion, maybe something like "Added support for Tiered Storage by enabling the configuration of custom storage plugins through the Kafka custom resource".